src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -108,7 +108,17 @@ public IActionResult Invoke(string name, [FromBody] FunctionInvocation invocatio
             {
                 { inputParameter.Name, invocation.Input }
             };
-            Task.Run(() => scriptHost.CallAsync(function.Name, arguments));
+
+            Task.Run(async () =>
+            {
+                using (_logger.BeginScope(new Dictionary<string, object>
```

Comment:
nit: formatting/indentation here is making things a bit confusing. Perhaps declare this dictionary before the call to `BeginScope`